tracing/README.md

```diff
@@ -69,13 +70,17 @@ _n_ new items were taken from a queue, and so on.
 to indicate that some code takes place within the context of that `Span`:
 
 ```rust
+# #[macro_use] extern crate tracing;
```

Comment:
@yaahallo we should now be using Rust 2018, so i think this, at least, can be removed.